conf/vagrant.yml

```diff
@@ -14,6 +14,11 @@
    - { role: drupal-db, tags: [ 'drupal-db' ] }
    - { role: letsencrypt, tags: [ 'letsencrypt' ] }
    - { role: sslterminator, tags: [ 'sslterminator' ] }
+# Optional HHVM role.
```

Comment:
Add notification that Wundermachina needs to be pointed to experimental hhvm branch for this to work.